nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestProcessorLifecycle.java

```diff
@@ -191,6 +193,7 @@ public void validateStopCallsAreMeaninglessIfProcessorNotStarted() throws Except
      * operations can only be @OnScheduled, @OnUnscheduled, @OnStopped.
      */
     @Test
+    @Category(IntegrationTest.class)
```

Comment:
These are the tests that deal with concurrency which is hard to tune when are running under load of full build. So these annotation will exclude them from full build yet they are still valid JUnit tests and will run under JUnit as before essentially sparing you from commenting/uncommenting @Ignore.